documentation/src/docs/metrics.md

```diff
@@ -7,491 +7,466 @@ You may obtain a copy of the License at
-```
-
-- Segment Store cache read/write metrics (counters):
-```
-segmentstore.cache.write_bytes
-segmentstore.cache.read_bytes
-```
-
-- Segment Store cache size (gauge) and generation spread (histogram) metrics:
-```
-segmentstore.cache.size_bytes
-segmentstore.cache.gen
-```
-
-- Tier-1 DurableDataLog read/write latency and queueing metrics (histogram):	
-```
-segmentstore.bookkeeper.total_write_latency_ms
-segmentstore.bookkeeper.write_latency_ms
-segmentstore.bookkeeper.write_queue_size
-segmentstore.bookkeeper.write_queue_fill
-```
-
-- Tier-1 DurableDataLog read/write (counter) and per-container ledger count metrics (gauge):	
-```
-segmentstore.bookkeeper.write_bytes
-segmentstore.bookkeeper.bookkeeper_ledger_count - with tag {"container", $containerId}
-```
-
-- Tier-2 Storage read/write latency metrics (histogram):	
-```
-segmentstore.storage.read_latency_ms
-segmentstore.storage.write_latency_ms
-```
-
-- Tier-2 Storage read/write data and file creation metrics (counters):
-```
-segmentstore.storage.read_bytes
-segmentstore.storage.write_bytes
-segmentstore.storage.create_count
-```
-
-- Segment Store container-specific operation metrics:
-```
-// Histograms - all with tags {"container", $containerId}
-segmentstore.container.process_operations.latency_ms
-segmentstore.container.process_operations.batch_size
-segmentstore.container.operation_queue.size
-segmentstore.container.operation_processor.in_flight
-segmentstore.container.operation_queue.wait_time
-segmentstore.container.operation_processor.delay_ms
-segmentstore.container.operation_commit.latency_ms
-segmentstore.container.operation.latency_ms
-segmentstore.container.operation_commit.metadata_txn_count
-segmentstore.container.operation_commit.memory_latency_ms
-
-// Gauge
-segmentstore.container.operation.log_size
-```
-
-- Segment Store operation processor (counter) metrics - all with tags {"container", $containerId}
-```
-// Counters/Meters
-segmentstore.container.append_count
-segmentstore.container.append_offset_count
-segmentstore.container.update_attributes_count
-segmentstore.container.get_attributes_count
-segmentstore.container.read_count
-segmentstore.container.get_info_count
-segmentstore.container.create_segment_count
-segmentstore.container.delete_segment_count
-segmentstore.container.merge_segment_count
-segmentstore.container.seal_count
-segmentstore.container.truncate_count
-
-```
-
-- Segment Store active Segments (gauge) and thread pool status (histogram) metrics:
-```
-// Gauge - with tags {"container", $containerId}
-segmentstore.active_segments
-
-// Histograms
-segmentstore.thread_pool.queue_size
-segmentstore.thread_pool.active_threads
-```
+- Segment Store Read/Write latency of storage operations ([Histograms](https://micrometer.io/docs/concepts#_histograms_and_percentiles)):
+
+   ```
+    pravega_segmentstore.segment.create_latency_ms
+    pravega_segmentstore.segment.read_latency_ms
+    pravega_segmentstore.segment.write_latency_ms
+
```

Comment:
@kevinhan88 added only the prefix pravega_...please check

Reply:
That's why we need a section talking about naming conversation:
1. prefix is configurable, so it may not be always "pravega"
2. "pravega_" -> "pravega."

Reply:
So you may call it out: prefix is omitted from the literal value of the metrics names in this doc (because the prefix is configurable), plus it is more readable without repeating the prefix everywhere.

Reply:
ok